en/flight_modes/mission.md

```diff
@@ -15,11 +15,11 @@
 
 Missions are usually created in a ground control station (e.g. [QGroundControl](https://docs.qgroundcontrol.com/en/PlanView/PlanView.html)) and uploaded prior to launch. They may also be created by a developer API, and/or uploaded in flight. Individual mission commands are handled in a way that is appropriate for each vehicle's flight characteristics (for example loiter is implemented as *hover* for copter and *circle* for fixed-wing). VTOL vehicles follow the behavior and parameters of fixed-wing when in FW mode, and of copter when in MC mode.
 
-> **Note** Missions are uploaded on SD card which need to be mounted/inserted **before** booting up the autopilot. 
+> **Note** Missions are uploaded on SD card which needs to be mounted/inserted **before** booting up the autopilot. 

```

Comment:
Good. Further improvement: "Missions are uploaded onto a SD card that needs to be inserted ..."